edx_toggles/tests/test_waffle.py

```diff
@@ -29,6 +30,29 @@ def test_constructor(self):
         self.assertEqual("module1", waffle.module_name)
         self.assertEqual(1, len(NaiveWaffle.get_instances()))
 
+    def test_toggle_methods_with_enabled_waffle(self):
+        waffle = NaiveWaffle("namespaced.name", "module1")
+        # test is_enabled method
+        self.assertEqual(True, waffle.is_enabled())
+        # test is_disabled method
+        self.assertEqual(False, waffle.is_disabled())
+        # test is_toggle_on method
```

Comment:
@Theo-flux: This comment is out of sync. Rather than updating it, I think the assertions are clear enough as-is and you could simply delete all of these `# test ...` comments from these two tests.

Reply:
@Theo-flux? 😄

Reply:
@Theo-flux: I'd love to have you be able to land this. Checking in one more time. Otherwise - I may end up taking this over at some point. Just let me know. Thanks.